cpp/ql/src/Critical/OverflowCalculated.ql

```diff
@@ -36,12 +35,12 @@ predicate spaceProblem(FunctionCall append, string msg) {
     malloc.getAllocatedSize() = add and
     buffer.getAnAccess() = strlen.getStringExpr() and
     (
-      insert.getTarget().hasGlobalName("strcpy") or
+      insert.getTarget().hasGlobalOrStdName("strcpy") or
       insert.getTarget().hasGlobalName("strncpy")
```

Comment:
`strncpy` is also available in `std`: https://en.cppreference.com/mwiki/index.php?title=Special%3ASearch&search=strncpy